tools/windows_portable/build_update.ps1

```
@@ -60,9 +60,24 @@ function Get-ExpectedRequirementsFingerprint([string]$ResolvedBundleDir) {
 
 function New-ManifestFiles([string]$SourceRoot, [string]$PayloadRoot) {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'New-ManifestFiles' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)